vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh

```diff
@@ -151,6 +151,9 @@ installFIPS() {
         retrycmd_if_failure 120 5 25 apt-mark hold ${image} || exit 1
     done
 
+    echo "purging unattended-upgrades in FIPS VHD..."
```

Comment:
@palma21 @justindavies FYI: as we talked in person in office today, remove/disable unattended-upgrades in FIPS VHDs